docs/INSTALL.md

```diff
@@ -37,29 +47,87 @@ $ gcloud auth login
 
-To apply the configuration file, you can use this command below.
+**Note**: Master DC/OS version is not meant for production use. It is only for CI/CD testing.
 
-### `cluster_profile.tfvars`
+To apply the configuration file and you accept all the default variables you can use this command below, otherwise you can continue along with this guide.
 
 ```bash
-# or similar depending on your environment
-echo 'public_ssh_key_path = "~/.ssh/id_rsa.pub"' >> cluster_profile.tfvars
-# lets set the clustername
-echo 'cluster_name = "my-ee-cluster"' >> cluster_profile.tfvars
-# we at mesosphere have to tag our instances with an owner and an expire date.
-echo 'tags = ["prod", "staging", "kubernetes"]' >> cluster_profile.tfvars
-# we have to explicitly set the version.
-echo 'dcos_version = "1.10.8"' >> cluster_profile.tfvars
-# we can set the azure location
-echo 'region = "us-west1"' >> cluster_profile.tfvars
-# set the google project id
-echo 'project_id = "massive-bliss-781"' >> cluster_profile.tfvars
+terraform apply
+```
+
+## Advanced YAML Configuration
+
+We have designed this project to be flexible. Here are the example working variables that allows very deep customization by using a single `main.tf` file.
+
+For advanced users with stringent requirements, here are DC/OS flag examples you can simply paste in `main.tf` file.
+The default variables inputs are tracked in the [terraform-gcp-dcos](https://registry.terraform.io/modules/dcos-terraform/dcos/gcp) terraform registry.
 
+```bash
+$ cat main.tf
+...
+module "dcos" {
+  source = "dcos-terraform/dcos/gcp"
+
+  # additional example variables in the module
+  dcos_version = "1.11.5"
+  dcos_instance_os = "centos_7.3"
+  num_masters = "3"
+  num_private_agents = "2"
+  num_public_agents = "1"
+  dcos_cluster_name = "DC/OS Cluster"
+  dcos_cluster_docker_credentials_enabled =  "true"
+  dcos_cluster_docker_credentials_write_to_etc = "true"
+  dcos_cluster_docker_credentials_dcos_owned = "false"
+  dcos_cluster_docker_registry_url = "https://index.docker.io"
+  dcos_use_proxy = "yes"
+  dcos_http_proxy = "example.com"
+  dcos_https_proxy = "example.com"
+  dcos_no_proxy = <<EOF
+  # YAML
+   - "internal.net"
+   - "169.254.169.254"
+  EOF
+  dcos_overlay_network = <<EOF
+  # YAML
+      vtep_subnet: 44.128.0.0/20
+      vtep_mac_oui: 70:B3:D5:00:00:00
+      overlays:
+        - name: dcos
+          subnet: 12.0.0.0/8
+          prefix: 26
+  EOF
+  dcos_rexray_config = <<EOF
+  # YAML
+    rexray:
+      loglevel: warn
+      modules:
+        default-admin:
+          host: tcp://127.0.0.1:61003
+      storageDrivers:
+      - ec2
+      volume:
+        unmount:
+          ignoreusedcount: true
+  EOF
+  dcos_cluster_docker_credentials = <<EOF
+  # YAML
+    auths:
+      'https://index.docker.io/v1/':
+        auth: Ze9ja2VyY3licmljSmVFOEJrcTY2eTV1WHhnSkVuVndjVEE=
+  EOF
 ```
+**Note**: The YAML comment is required for the DC/OS specific YAML settings.
```

Comment:
<p class="message--note"><strong>NOTE: </strong>The YAML comment is required for the DC/OS specific YAML settings.</p>

Reply:
Remove the content in line #165 and copy-paste the above line.